widget/app.js

```diff
@@ -72,7 +72,7 @@
                                                         });
                                                     }
                                                     if (res) {
-                                                        res = res.filter(item=>(!(item.mediaType==='audio' && (item.originalMediaUrl.includes("www.dropbox") || item.originalMediaUrl.includes("dl.dropbox")) && !item.dropboxAudioUpdated)))
+                                                        res = res.filter(item=>(!(item.mediaType==='audio' && (item.originalMediaUrl.includes("www.dropbox") || item.originalMediaUrl.includes("dl.dropbox")) && !item.dropboxAudioUpdatedV2)))
```

Comment:
- Keep the old settings
- just add new one `audioDownloadUpdated` as new filter condition under this block

`res.filter(item=>( // ios  and fileType audio)`